### PR TITLE
Omd monitoring

### PR DESCRIPTION
**Pull request by vpramo**
This commit enables OMD Check MK Monitoring for VPC cluster

### CODE REVIEW

**Comment by vpramo** [comment]
@sajuptpm please review and merge

File: site.pp [hunk 1]
```diff
@@ -29,12 +35,17 @@
...
+  include rjil::omd_client
+  include rjil::monitoring::base
+  include rjil::monitoring::vpcctrl
 }
 
 
 
 node /^cp\d+/ {
   include rjil::contrail::vrouter_standalone
+  include rjil::omd_client
```
Comment (sajuptpm) [inline site.pp]: compute team don't use consul
So we can't include rjil::monitoring::base in CP

Comment (vpramo) [inline site.pp]: Yes, because the deployment checks we are getting from consul

File: manifests/omd_client.pp [hunk 2]
```diff
@@ -0,0 +1,9 @@
+class rjil::omd_client (
```
Comment (sajuptpm) [inline manifests/omd_client.pp]: How omd_clients get IP of omd_server (vpc_monitor) ?.

Comment (vpramo) [inline manifests/omd_client.pp]: They dont, We have to add the IP in the server and then they establish connection

File: manifests/omd_server.pp [hunk 3]
```diff
@@ -0,0 +1,7 @@
+class rjil::omd_server {
+
+  include ::omd::server
+
+}
+
+
```
Comment (sajuptpm) [inline manifests/omd_server.pp]: Note: right now, we are configuring omd_clients IPs in omd_server via omd_server GUI.

**Pull request merged**